mito-ai/src/Extensions/AiChat/ChatHistoryManager.tsx

```diff
@@ -264,6 +264,25 @@ export class ChatHistoryManager {
         );
     }
 
+    addStreamingAIMessage(
+        messageContent: string,
+        promptType: PromptType
+    ): void {
+        // Find the last AI message in the history
+        const lastAIMessageIndex = this.getLastAIMessageIndex();
```

Comment:
So we are making an assumption here that there are never going to be an assistant message followed directly by another assistant message... that seems reasonable! Want to just document that. If that invariant changes, then we can just start associated ID's with the messages and checking it that way.